platform-includes/getting-started-config/javascript.bun.mdx

```diff
@@ -1,16 +1,25 @@
 <SignInNote />
 
-```javascript {"onboardingOptions": {"performance": "5-9"}}
+Sentry should be initialized as early in your app as possible. It is essential that you call `Sentry.init` before you require any other modules in your application—otherwise, auto-instrumentation of these modules will _not_ work.
+
+Once this is done, Sentry's Bun SDK captures unhandled exceptions as well as tracing data for your application.
+
+You need to create a file named `instrument.js` that imports and initializes Sentry:
+
+```javascript {filename: instrument.js} {"onboardingOptions": {"performance": "5-9"}}
```

Comment:
```suggestion
```javascript {filename: instrument.js} {"onboardingOptions": {"performance": "7-11"}}
```
I think this needs to be bumped because you added an extra line and the comment to the snippet.